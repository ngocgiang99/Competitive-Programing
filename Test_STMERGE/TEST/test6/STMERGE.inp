5
5 6
677231991 937188136 313083815 12468301 945928668 208928855 
59535126 107954457 801833885 778959597 436796283 954416943 
774985471 667655360 396432187 711415940 101952647 166557156 
918060797 821941068 411666171 657629358 164368804 4253182 
900610398 521237428 60764245 162057550 854983415 240316363 
6 8
151461706 254814214 119099308 978609626 491367452 314747324 141100508 418212801 
289114482 328243330 933305130 157821339 806094225 146289196 896157692 498149722 
944086271 774078671 721840374 134631222 466584191 920107713 21755646 702604707 
13628184 612216982 724393273 629566294 185634830 939446563 135276299 297225027 
909883110 702049150 218265114 859717793 11970917 147509499 840996427 355640035 
869374865 740269360 554926530 691893695 920639405 825306097 496684261 184093482 
10 5
902071896 61947225 633717261 188609452 607129857 
10059969 340902939 60985744 722757637 53692951 
132845486 371240405 736591561 80166991 177035339 
301027727 51120369 216192672 848115640 532108307 
291229283 768170109 872411128 485493495 422657347 
776920477 405925202 463355919 722281975 379736051 
718274633 458264945 706204452 536443317 765548452 
709743473 273657774 582253912 882270311 254104740 
198016699 26878353 907356631 974936544 764134831 
505605973 311277434 617506361 145282097 511170520 
1 10
612795939 992454289 477616297 979567063 201510829 398166323 539464831 507307142 16804146 999518133 
10 3
981397336 364287050 328775492 
477114382 527990531 428466024 
809624572 171580368 867861337 
652846020 42043529 698927939 
96580543 709142611 687615601 
824352123 131232217 188686410 
527568247 385602510 705467789 
823650589 88132002 398109648 
777791232 915935363 573965622 
109028182 638578326 49801327 
