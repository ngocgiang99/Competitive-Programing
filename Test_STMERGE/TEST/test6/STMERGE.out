118674130
325193990
187095308
524111288
109028182
